effectiveOfficeBackend/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM gradle:7-jdk11 AS build
+COPY . /home/gradle/src
+WORKDIR /home/gradle/src
+RUN ./gradlew buildFatJar --no-daemon
+
+FROM openjdk:11
+EXPOSE 8080:8080
+WORKDIR /app
+COPY --from=build /home/gradle/src/build/libs/*.jar /app/app.jar
+ENTRYPOINT ["java","-jar","/app/app.jar"]
+# to build: docker build -t effective_office_backend .
```

Comment:
Also prefer to left comments about working with Docker image (i. e. build, run, etc.) in README.md